### PR TITLE
Use creation facility ID instead of facility ID for recent patients

### PR DESCRIPTION
**Pull request by rakshakhegde**
https://www.pivotaltracker.com/story/show/170485623

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/patient/RecentPatient.kt [hunk 1]
```diff
@@ -56,7 +56,7 @@ data class RecentPatient(
           LEFT JOIN (
             SELECT MAX(createdAt) latestCreatedAt, uuid, patientUuid, facilityUuid
               FROM Appointment
-              WHERE facilityUuid = :facilityUuid
+              WHERE creationFacilityUuid = :facilityUuid
```
Comment (sanchita) [inline app/src/main/java/org/simple/clinic/patient/RecentPatient.kt]: Please add a test for this.

**Comment by sanchita** [comment]
Moving this to WIP as the query is not correct.
We should update the `WHERE` clause to `AP.creationFacilityUuid = :facilityUuid`

**Comment by rakshakhegde** [comment]
`creationFacilityUuid` that is being used is that of `Appointment`.

File: app/src/main/java/org/simple/clinic/patient/RecentPatient.kt (hunk 1 above)
Comment (rakshakhegde) [reply app/src/main/java/org/simple/clinic/patient/RecentPatient.kt]: done

**Pull request merged**